helm/values-prod.yaml

```
@@ -0,0 +1,21 @@
+common:
+  secrets:
+    GITHUB_ACCESS_TOKEN: ref+vault://kv/cicd/gitlab/parity/opstooling/pr-custom-review/parity-prod#DB_PASSWORD
```

```suggestion
    GITHUB_ACCESS_TOKEN: ref+vault://kv/cicd/gitlab/parity/opstooling/pr-custom-review/parity-prod#GITHUB_ACCESS_TOKEN
```

I think it's better to keep names consistent.